index.html

```diff
@@ -23,7 +23,7 @@
     <!-- Navigation -->
     <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
       <div class="container">
-        <a class="navbar-brand" href="https://epiclab.github.io/">Sarma Research Lab</a>
+        <a class="navbar-brand" href="https://epiclab.github.io/">EPIC Lab</a>
```

Comment:
Change to `EPICLab` for all places that you updated to `EPIC Lab`, so that it's consistent across all of our platforms (Slack, Box, Google Drive, etc.).